### PR TITLE
text and assets for the section technologies

### PR DESCRIPTION
**Pull request by radeklawgmin**
Section on technologies used in Websight

## Description
Text and assets ilustrating WS technology stack.

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue) labeled with `bug`
- [x] New feature (non-breaking change which adds functionality) labeled with `enhancement`
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist:

- [x] My code follows the [code standards](/CONTRIBUTING.md) of this project.
- [ ] My change requires updating the documentation. I have updated the documentation accordingly.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -18,6 +18,73 @@ and then see the results on [localhost:8080/apps/websight/index.html/content::sp
...
+            <td width="100" height="70"></td>
+        </tr>
+        <tr>
+            <td width="100" height="70"></td>
+            <td width="100" height="70"></td>
+            <td width="100" height="70"></td>
+            <td colspan="2" width="100" height="70"><center><a href="https://yaml.org"><img src="/assets/technologies/yaml-logo.png" alt="YAML" style="width:50px;" align="center" /></a></center></td>
+            <td width="100" height="70"></td>
+        </tr>
+        </style>
```
Comment (paweljozwicki) [inline README.md:83]: It's an unnecessary closing style tag

File: README.md [hunk 2]
```diff
@@ -18,6 +18,73 @@ and then see the results on [localhost:8080/apps/websight/index.html/content::sp
...
+     <tr>
+     <thead>
+        <tr>
+            <th colspan="2" style='width: 320px'><center>CMS</center></th>
+            <th style='width: 160px'><center>Runtime</center></th>
+            <th colspan="2" style='width: 320px'><center>Supportive</center></th>
+            <th style='width: 160px'><center>Design</center></th>
+        </tr>
+    </thead>
+    <tbody>
```
Comment (paweljozwicki) [inline README.md:44]: There is a missing `<tr>`

File: README.md [hunk 3]
```diff
@@ -18,6 +18,73 @@ and then see the results on [localhost:8080/apps/websight/index.html/content::sp
...
+            <td width="100" height="70"><a href="https://www.java.com"><img src="/assets/technologies/java-logo.png" alt="Java" style="width:80px;" align="right"/></a></td>
+            <td width="100" height="70"><a href="https://felix.apache.org"><img src="/assets/technologies/osgi-logo.png" alt="OSGi" style="width:90px;" align="left" /></a></td>
+            <td width="100" height="70"><center><a href="https://aws.amazon.com"><img src="/assets/technologies/aws-logo.png" alt="AWS" style="width:60px;" align="center" /></a></center></td>
```
Comment (paweljozwicki) [inline README.md:47]: `width`, `height`, `align` attributes and `<center>` tag are deprecated. Maybe we could change them with style="width: 100px; height: 70px; text-align: center"?

**Comment by radeklawgmin** [comment]
It's going to be addressed with Markdown in a separate PR.

**Pull request closed without merging**